pkg/agent/controller/networkpolicy/cache_test.go

```diff
@@ -619,8 +627,20 @@ func TestRuleCacheAddNetworkPolicy(t *testing.T) {
 			UID:       "policy2",
 		},
 	}
+	networkPolicy3 := &v1beta2.NetworkPolicy{
+		ObjectMeta:      metav1.ObjectMeta{UID: "policy3", Namespace: "ns3", Name: "name3"},
+		Rules:           []v1beta2.NetworkPolicyRule{*networkPolicyRule3},
+		AppliedToGroups: nil,
```

Comment:
I'm curious as to why `AppliedToPerRule` is not set to true in this object?

Reply:
My bad. Updated the testcase to reflect what it should look like after networkpolicy controller processing.